### PR TITLE
Append cdap group to hadoop.proxyuser.hive.groups

### PR DESCRIPTION
**Pull request by wolf31o2**

### CODE REVIEW

File: attributes/security.rb [hunk 1]
```diff
@@ -65,6 +65,18 @@
...
+  # Add cdap group to core-site.xml's hadoop.proxyuser.hive.groups
+  if node['hadoop'].key?('core_site') && node['hadoop']['core_site'].key?('hadoop.proxyuser.hive.groups')
+    arr = node['hadoop']['core_site']['hadoop.proxyuser.hive.groups'].split(',')
+    group = 'cdap'
+    unless arr.include?(group)
+      arr += [group]
+      default['hadoop']['core_site']['hadoop.proxyuser.hive.groups'] = arr.join(',')
+    end
+  else
+    default['hadoop']['core_site']['hadoop.proxyuser.hive.groups'] = 'cdap,hadoop'
```
Comment (jwang47) [inline attributes/security.rb]: No need for `hive` user?

Comment (wolf31o2) [inline attributes/security.rb]: This is groups, not users. The `hive` user is a member of `hadoop` just like the other service accounts.

Comment (wolf31o2) [inline attributes/security.rb]: Also, the default set in caskdata/hadoop_wrapper_cookbook#51 has hive in that list, so we'll append to it in practice. This is for people not necessarily running Hive to not have their security mucked with by the cookbook.

**Comment by jwang47** [comment]
LGTM

**Pull request merged**